app/src/main/java/com/example/practiceapp/RecyclerViewFragHomeProvider.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.practiceapp
+
+class RecyclerViewFragHomeProvider {
```

Comment:
the naming is not clear, why is this related to REcyclerViewFragHome? you should have only one list that I think you already have in another provider